### DISCUSSION TITLE
Question - UIScreen.main.bounds is including status bar?

### DISCUSSION THREAD

**Discussion by frodriguez-hu** [luicfrr/react-native-vision-camera-face-detector#84] [Q&A] [answered]
Hey @luicfrr I am wondering if the function you are using to get the `height` are including the status bar and if it is something that maybe we would like to set up optionally, what do you think?

I am wondering it for ios and android

**Comment by luicfrr** [comment, marked as answer]
fixed in [81ff6de](https://github.com/luicfrr/react-native-vision-camera-face-detector/commit/81ff6de5a7e9263e97e79265b0475b6afea95bac)

**Reply by frodriguez-hu** [reply]
Great!!!!! Thanks for that, I didn't think on bringing it on the options. Let me know if I can help you to anything to improve the library, thanks!
